int8 nAnchors
anchor[10] anchors